packages/non-core/facebook/deprecation_notice.js

```diff
@@ -0,0 +1,9 @@
+console.warn(
```

Comment:
Ah, now I remember I moved them on the https://github.com/meteor/meteor/tree/release-1.4.3 branch but not `devel`.